README.md

```diff
@@ -24,20 +24,28 @@ These are organizations of DFKI groups and their (notable) open-source projects
 		- [Agora Ecosystem](https://github.com/agora-ecosystem) - A Data Science Marketplace infrastructure project from DIMA (TU- Berlin) and IAM (DFKI)
 
 	- DFKI Site Bremen - DFKI Standort Bremen
+	        
 		- [DFKI RIC](https://github.com/dfki-ric) - DFKI's Robotics Innovation Center
+		  	- [List of maintianed OSS projects by DFKI RIC](https://dfki-ric.github.io/)
 			- [ROCK](https://www.rock-robotics.org/) - the Robot Construction Kit
 			- [Alexander Fabisch](https://github.com/AlexanderFabisch)
-				- [Gaussian Mixture Regression](https://github.com/AlexanderFabisch/gmr)
+			- [Gaussian Mixture Regression](https://github.com/AlexanderFabisch/gmr)
```

Comment:
Hm, was the removal of the indentation intended?

Reply:
no